cmake/fficonfig.h.in

```diff
@@ -28,14 +28,14 @@
 /* #undef FFI_NO_STRUCTS */
 
 /* Define to 1 if you have `alloca', as a function or macro. */
-#define HAVE_ALLOCA 1
+#cmakedefine HAVE_ALLOCA 1
```

Comment:
Missing configure test.
Could be added after this line: https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/blob/7812f43ab5cbe2840809997d5d8bbc8b5abd9989/cmake/ConfigureChecks.cmake#L679